### DISCUSSION TITLE
unable to find the argocd-initial-admin-secret when upgraded to v2.0.4

### DISCUSSION THREAD

**Discussion by kannamr** [argoproj/argo-cd#6786] [Q&A]
Hi,
    I have upgarded argo to v2.0.4. The last version I used was v1.7. I see after v1.9, the pod name is no longer admin password. From the docs, argocd-initial-admin-secret has be in the secrets. But unfortunately, I couldn't find the secret when using ```kubectl get secret -n argocd``` Resetting the password works, but I don't want to reset and I would like to use the initial password itself. Can someone help me with the issue.

**Comment by jgwest** [comment]
Hi @kannamr, if you create a new installation of Argo CD, you should see `argocd-initial-admin-secret` secret. This is the case for v2.0.x, and I just verified it continues to be case for the upcoming v2.1.x.

On the other hand if you upgrade from an existing install, I expect this secret will not be created. I'm not aware of a way to 'reset' this state, for example, if you wanted this secret to be created on an existing install.

**Reply by kannamr** [reply]
@jgwest thanks for the response. So I need a new install in order for the secret to be available. So the only option is to change admin password once upgraded. Right?

**Reply by kannamr** [reply]
@jgwest In that case is there a way I can get the ```admin.password``` from ```argocd-secret```. The base64 decode will not work.

**Reply by jgwest** [reply]
My understanding is that that should work, once the `argocd-initial-admin-secret` is deleted: https://argoproj.github.io/argo-cd/getting_started/#4-login-using-the-cli

**Reply by tomikonio** [reply]
@kannamr The base64 decode of the `admin.password` returns a hashed string of your password.
This string **cannot** be decrypted.
Do you know the password that you used before the upgrade ? It should not have been changed (unless you were using the name of the argocd-server pod as password).

If it does not work then the solution is to change the password as noted [here](https://argoproj.github.io/argo-cd/faq/#i-forgot-the-admin-password-how-do-i-reset-it) .
